packages/server/graphql/mutations/createGitHubTaskIntegration.ts

```diff
@@ -68,44 +71,77 @@ export default {
       })
     }
 
-    // RESOLUTION
     const {content: rawContentStr, meetingId} = task
-    const viewerAuth = await dataLoader.get('githubAuth').load({teamId, userId: viewerId})
-
-    if (!viewerAuth) {
+    const [viewerAuth, assigneeAuth, team] = await Promise.all([
+      dataLoader.get('githubAuth').load({teamId, userId: viewerId}),
+      userId && dataLoader.get('githubAuth').load({teamId, userId}),
+      dataLoader.get('teams').load(teamId)
+    ])
+    const auth = assigneeAuth ?? viewerAuth
+    if (!auth) {
       return standardError(
-        new Error(`Assignment failed! The assignee does not have access to GitHub`),
+        new Error(`Assignment failed! Neither you nor the assignee has access to GitHub`),
         {userId: viewerId}
       )
     }
+    const teamMembers = await dataLoader.get('teamMembersByTeamId').load(teamId)
+    const viewer = teamMembers.find(({userId}) => userId === viewerId)
```

Comment:
nit-- we can call this viewerTeamMember since viewer usually refers to a User object